go.mod

```diff
@@ -32,11 +36,13 @@ require (
 	k8s.io/component-base v0.0.0
 	k8s.io/klog v0.3.2
 	k8s.io/kube-openapi v0.0.0-20190401085232-94e1e7b7574c // indirect
+	k8s.io/kubelet v1.15.0
 	k8s.io/kubernetes v1.15.0
 	k8s.io/sample-controller v0.0.0-20190731144349-6f8905ae4ee5
 )
 
 replace (
+	gopkg.in/gcfg.v1 => github.com/mandelsoft/gcfg v1.2.4-0.20191118133708-71f203a139d3
```

Comment:
Is this replace needed still?

Reply:
There were several problems with the gcfg API. For example it does not support a generic map as section used to read the arbitrary tag list. I've provided a pull request go-gcfg/gcfg#18 for this project but nobody is reaction. 

If the controller manager switches as planned to a json config file format, we would get rid of this problem at all.

May be it is acceptable to use this fork version in between.

Reply:
I see, and this is needed for the tags feature?

Reply:
we replaced it by a `tags` attribute using a JSON map as string value